cloudferrylib/os/compute/nova_compute.py

```diff
@@ -89,8 +89,8 @@ def _read_info_resources(self, **kwargs):
         for keypair in self.get_keypair_list():
             info['keypairs'][keypair.id] = self.convert(keypair)
 
-        for flavor in self.get_flavor_list():
-            info['flavors'][flavor.id] = self.convert(flavor)
+        for flavor in self.get_flavor_list(is_public=None):
```

Comment:
1. This looks counter-intuitive. `is_public` looks like boolean variable, while for some reason you assign it to `None` rather than `True` or `False`
2. `get_flavor_list` is always used with `is_public=None` throughout the code, why not setting it directly in the `get_flavor_list()`? In other words instead of modifying all the calls to `get_flavor_list`, just modify the internals of the method.